src/open_inwoner/templates/cms/fullwidth.html

```diff
@@ -1,11 +1,21 @@
 {% extends 'master.html' %}
 {% load i18n cms_tags sekizai_tags static %}
 
+{% block mount_modules %} data-mount-modules='["sidenav"]'{% endblock %}
+
+{% block react_modules %}
+{# Including this script will allow the page to use the react router #}
```

Comment:
React router? i think you meant to use react components.

```suggestion
{# Including this script will allow the page to use react components #}
```

Reply:
Agreed, I'd really like to avoid using `router` for our `loader` because it's ambiguous with the widely used [react-router libary](https://reactrouter.com/).

Reply:
I put that 'router' text there because I copied it from Stefan's src/open_inwoner/templates/demo/ReactDemo.html template, for our portal.

Let me know if I should change both texts 😄